### PR TITLE
add the image input field. (#27)

### PR DESCRIPTION
**Pull request by ihab4real**
issues: #27 

image

### CODE REVIEW

File: lib/pages/user_form_page.dart [hunk 1]
```diff
@@ -27,6 +30,8 @@ class _UserFormPageState extends State<UserFormPage> {
   String errorMsg = "";
   String username = "", email = "", password = "", confirmPassword = "";
 
+  File? campaignImage;
```
Comment (Yassa-hue) [inline lib/pages/user_form_page.dart:33]: campaignImage !! WTF ?!

**Review by Yassa-hue — CHANGES_REQUESTED** [review]
I don't like the UX of this section:

image


The icon on the top and then the image section looks off.

Here are two suggestions (you can pick any, it's up to you):
* Move the image widget to the bottom of the screen.
* Change the shape of the image to be a rectangle like the other input fields.

File: lib/pages/user_form_page.dart (hunk 1 above)
Comment (ihab4real) [reply lib/pages/user_form_page.dart]: anything else? (really thought?)

**Comment by ihab4real** [comment]
> * Move the image widget to the bottom of the screen.

i did this one. it got even weirder. but we need the round thing to give the user a preview of the image after being cropped.

File: lib/pages/user_form_page.dart (hunk 1 above)
Comment (Yassa-hue) [reply lib/pages/user_form_page.dart]: userProfileImage ?

**Comment by Yassa-hue** [comment]
As discussed on the DM, use the layout that I proposed and merge.
@ihab4real

**Pull request closed without merging**